python-sdk/src/astro/airflow/datasets.py

```diff
@@ -0,0 +1,9 @@
+try:
```

Comment:
Are we planning to introduce more things inside the package `airflow` inside Astro?

Since this is a configuration `DATASET_SUPPORT`), to some extent, have you considered adding this to `settings.py` or `constants.py` (`AIRFLOW_DATASET_SUPPORT`)?

Reply:
Yeah I am thinking of adding `kwargs_with_datasets` --> same as https://github.com/astronomer/astro-sdk/blob/aip-48-mapping/src/astro/utils/compat.py